### PR TITLE
update mailmap for 0.23

### PR DESCRIPTION
**Pull request by agramfort**

### CODE REVIEW

File: .mailmap [hunk 1]
```diff
@@ -202,4 +203,27 @@ Tanay Gahlot <tanaygahlot@gmail.com> Tanay <tanaygahlot@gmail.com>
...
+Enrico Vara <69973551+enricovara@users.noreply.github.com> enricovara <69973551+enricovara@users.noreply.github.com>
+Giorgio Marinato <giorgio.marinato@unitn.it> neurogima <76406896+neurogima@users.noreply.github.com>
+Erica Peterson <nordme@uw.edu> nordme <nordme@uw.edu>
+Erica Peterson <nordme@uw.edu> nordme <38704848+nordme@users.noreply.github.com>
+Timon Merk <38216460+timonmerk@users.noreply.github.com> timonmerk <38216460+timonmerk@users.noreply.github.com>
+apadee <44297909+apadee@users.noreply.github.com> apadee <44297909+apadee@users.noreply.github.com>
+kimcoco <41998428+kimcoco@users.noreply.github.com> kimcoco <41998428+kimcoco@users.noreply.github.com>
+monkeyman192 <monkey_man_192@yahoo.com.au> monkeyman192 <monkey_man_192@yahoo.com.au>
+sfc-neuro <78911192+sfc-neuro@users.noreply.github.com> sfc-neuro <78911192+sfc-neuro@users.noreply.github.com>
+DataFox <sfox@riseup.net> DataFox <sfox@riseup.net>
```
Comment (agramfort) [inline .mailmap:229]: @larsoner or @drammock if you can help me get the true names of these people it would help

Comment (drammock) [reply .mailmap:229]: I'll push what I find here

Comment (drammock) [reply .mailmap]: done :)

**Review by agramfort — COMMENTED** [review]
Perfect 👌 merge when green

**Pull request merged**